scripts/change-log-generator-utils.js

```diff
@@ -263,7 +263,7 @@ function filterPackageNames(packageHeaders) {
  */
 function generateKey(packageName, type, packagesToIgnore) {
   if (
-    typesToIgnore.some(typeToIgnore => type.startsWith(typeToIgnore)) ||
+    typesToIgnore.some(typeToIgnore => !type || type.startsWith(typeToIgnore)) ||
```

Comment:
this is a work around to not fail to create the change log commit if we aren't able to parse a commit title.